text.html

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
```

Comment:
You should not put .html files into the repository. Just place such files under the tmp/articles directory and they will not be seen here. Check documentation for more details.